src/idlcxx/src/streamers.c

```diff
@@ -992,6 +960,47 @@ process_case_label(
   return IDL_RETCODE_OK;
 }
 
+static idl_retcode_t
+process_typedef_decl(
+  const idl_pstate_t* pstate,
+  struct streams* streams,
+  const idl_type_spec_t* type_spec,
+  const idl_declarator_t* declarator)
+{
+  static const instance_location_t loc = { .parent = "instance", .type = TYPEDEF | NORMAL_INSTANCE | KEY_INSTANCE };
```

Comment:
`process_instance` takes an `instance_location_t` rather than a `const instance_location_t`

Reply:
I will correct this.